docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md

```diff
@@ -93,97 +93,67 @@ The **Performance** tab provides the following metrics.
-| Block cache usage | A block requires multiple touches before it is added to the multi-touch (hot) portion of the cache.<br><br>Multi Touch: the size (in bytes) of the cache usage by blocks having multiple touches.<br>Single Touch: the size (in bytes) of the cache usage by blocks having only a single touch. | |
-| LSM-DB Blooms usefulness | Blooms checked: the number of times the bloom filter has been checked.<br>Blooms useful: the number of times the bloom filter has avoided file reads (avoiding iops). | Bloom filters are hash tables used to determine if a given sstable has the data for a query looking for a particular value.<br>Bloom filters are not helpful for range queries. |
-| Stalls | Time the writer has to wait for compactions or flushes to finish. | |
-| Rejections | The number of RPC requests rejected due to the number of majority SST files. | Rejections can happen due to hitting the soft limit on SST files, hitting more than the soft memory limit (set to 85% of the hard limit), and so on. |
-| Flush write | The number of bytes written during the flush process. | |
-| Compaction time (ms) | Time for the compaction processes to complete. | |
-| Compaction num files | The average number of files in any single compaction. | |
-| Transaction | Expired Transactions: the number of expired distributed transactions.<br>Transaction Conflicts: the number of conflicts detected among uncommitted distributed transactions.<br><br>This is related to the process that resolves conflicts for write transactions. This process reads all intents that could conflict and tries to abort transactions with a lower priority. If a write transaction conflicts with a higher-priority transaction, then an error is returned and this metric is iterated. | |
-| Transaction pool cache hits | Percentage of transaction pool requests fulfilled by the transaction pool cache. | |-->
-
-<!--### YSQL Advanced
-
-| Graph | Description | Use |
-| :---| :--- | :--- |
-| YSQL Operations/sec  | The number of various transaction control operations, such as BEGIN, COMMIT, ROLLBACK, as well as other operations through the YSQL API. | You may consider this information while examining other metrics. |
-| YSQL Average Latency (ms) | The average time taken by various transaction control operations, such as BEGIN, COMMIT, ROLLBACK, as well as other operations through the YSQL API. | You may consider this information while examining other metrics. |
--->
+### YB-Master server
 
-### Master server
+The [YugabyteDB Master Server](../../../architecture/concepts/yb-master/) (YB-Master) hosts system metadata, and records such as what tables exist in the system, where their tablets live, what users and roles exist, the permissions associated with them, and so on. YB-Masters are also responsible for coordinating background operations.
```

Comment:
The [YugabyteDB Master Server](../../../architecture/concepts/yb-master/) (YB-Master) hosts system metadata, and records such as what tables exist in the system, where their tablets live, what users and roles exist, the permissions associated with them, and so on.
Suggestion:
The [YugabyteDB Master Server](../../../architecture/concepts/yb-master/) (YB-Master) hosts system metadata, records about tables in the system and locations of their tablets, users, roles, permissions, and so on.